File: CS50x 2023/PSET06/sentimental-mario-more/mario.py
from cs50 import get_int

while True:
    height = get_int("Height: ")

    if height >= 1 and height <= 8:
        break
    else:
        continue

for i in range(height):
    print(" " * (height - (i + 1)) + "#" * (i + 1) + "  " + "#" * (i + 1))

File: CS50x 2023/PSET01/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    printf("\n************************ Mario Less ************************\n\n");

    // Get Height From User, as a Integer, Between 1 and 8
    int height = 0;

    printf("Enter Height (From 1 to 8):\n");
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    printf("\n");

    // Show Mario Blocks
    for (int i = 1; i <= height; i++)
    {
        // Show Empty Blocks
        for (int j = 0; j < height - i; j++)
        {
            printf(" ");
        }

        // Show Hashtag Blocks
        for (int k = 0; k < i; k++)
        {
            printf("#");
        }

        // Go To Next Row
        printf("\n");
    }
}